drafts/liberty-dev-mode-gradle.adoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Open Liberty development mode with Gradle"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/28316667
+author_github: https://github.com/rzgry
+seo-title: Open Liberty development mode with Gradle - OpenLiberty.io
+seo-description: Develop Open Liberty applications quicker using Open Liberty development mode with Gradle.
+blog_description: "Develop Open Liberty applications quicker using Open Liberty development mode with Gradle."
```

Comment:
"Open Liberty" is used too many times, perhaps change to `Develop Java applications quicker using Open Liberty development mode with Gradle.` for both SEO and blog description.